### DISCUSSION TITLE
Update globaleaks

### DISCUSSION THREAD

**Discussion by matteoperazzolo** [globaleaks/globaleaks-whistleblowing-software#4292] [General]
Good morning,
I ask you for information. We should update our globalaks do you have any recommendations for updating? What schedule do you recommend?

**Comment by evilaliv3** [comment]
Thank you for your question.

GlobaLeaks is always released considering stability and security as top priority so that users should not have particular doubts when upgrading.
At the moment we are releasing a large set of minor releases following a very major change related to the client but we expect to be concluding soon with a relese 5.0.X concluding the release 5.0 as final.

Following this we will proceed developing and publishing a more precise release schedule with few major release a year (2 to 4)  and minor releases issues only for security reasons.

Do you have any specific question ?

**Comment by matteoperazzolo** [comment]
Good morning,
we have thought of this update procedure:

  *   Machine snapshot
  *   Check with a manager n. open reports
  *   Update everything
  *   Re-login with a manager and check if he still sees everything and make a test report to close
Is it too much? Or can we do something lighter?

If I may I also wanted to ask you this question, has the possibility of giving some users the possibility of masking irrelevant data been added? And has the possibility of deleting irrelevant data within reports been added?
If so, I would like to understand how to configure it.

Thank you.
Have a nice day.

Matteo Perazzolo
***@***.******@***.***>

***@***.***<https://www.itstrategy.it/>
IT Strategy Srl con socio unico
Viale del Mercato Nuovo, 44/F - 36100 Vicenza
Tel. +39 0444 965587
www.itstrategy.it<http://www.itstrategy.it/>



***@***.***

Le informazioni contenute nel presente messaggio e nei relativi allegati sono riservate. Se non siete il destinatario sopra indicato, Vi invitiamo a darcene immediata comunicazione e a distruggere il presente messaggio e i relativi allegati.
The information contained in this email and any attachments are confidential. If you are not the intended recipient, please notify the sender immediately and destroy this email and any attachments.

Da: Giovanni Pellerano ***@***.***>
Inviato: martedì 5 novembre 2024 22:46
A: globaleaks/globaleaks-whistleblowing-software ***@***.***>
Cc: Matteo Perazzolo ***@***.***>; Author ***@***.***>
Oggetto: Re: [globaleaks/globaleaks-whistleblowing-software] Update globaleaks (Discussion #4292)


Thank you for your question.

GlobaLeaks is always released considering stability and security as top priority so that users should not have particular doubts when upgrading.
At the moment we are releasing a large set of minor releases following a very major change related to the client but we expect to be concluding soon with a relese 5.0.X concluding the release 5.0 as final.

Following this we will proceed developing and publishing a more precise release schedule with few major release a year (2 to 4) and minor releases issues only for security reasons.

Do you have any specific question ?

**Comment by matteoperazzolo** [comment]
Hello,
Do you have news?

Thanks.

Cordiali saluti,

Matteo Perazzolo
***@***.******@***.***>

***@***.***<https://www.itstrategy.it/>
IT Strategy Srl con socio unico
Viale del Mercato Nuovo, 44/F - 36100 Vicenza
Tel. +39 0444 965587
www.itstrategy.it<http://www.itstrategy.it/>



***@***.***

Le informazioni contenute nel presente messaggio e nei relativi allegati sono riservate. Se non siete il destinatario sopra indicato, Vi invitiamo a darcene immediata comunicazione e a distruggere il presente messaggio e i relativi allegati.
The information contained in this email and any attachments are confidential. If you are not the intended recipient, please notify the sender immediately and destroy this email and any attachments.

Da: Matteo Perazzolo
Inviato: mercoledì 6 novembre 2024 09:10
A: globaleaks/globaleaks-whistleblowing-software ***@***.***>; globaleaks/globaleaks-whistleblowing-software ***@***.***>
Cc: Author ***@***.***>
Oggetto: R: [globaleaks/globaleaks-whistleblowing-software] Update globaleaks (Discussion #4292)

Good morning,
we have thought of this update procedure:

  *   Machine snapshot
  *   Check with a manager n. open reports
  *   Update everything
  *   Re-login with a manager and check if he still sees everything and make a test report to close
Is it too much? Or can we do something lighter?

If I may I also wanted to ask you this question, has the possibility of giving some users the possibility of masking irrelevant data been added? And has the possibility of deleting irrelevant data within reports been added?
If so, I would like to understand how to configure it.

Thank you.
Have a nice day.

Matteo Perazzolo
***@***.******@***.***>

***@***.***<https://www.itstrategy.it/>
IT Strategy Srl con socio unico
Viale del Mercato Nuovo, 44/F - 36100 Vicenza
Tel. +39 0444 965587
www.itstrategy.it<http://www.itstrategy.it/>



***@***.***

Le informazioni contenute nel presente messaggio e nei relativi allegati sono riservate. Se non siete il destinatario sopra indicato, Vi invitiamo a darcene immediata comunicazione e a distruggere il presente messaggio e i relativi allegati.
The information contained in this email and any attachments are confidential. If you are not the intended recipient, please notify the sender immediately and destroy this email and any attachments.

Da: Giovanni Pellerano ***@***.******@***.***>>
Inviato: martedì 5 novembre 2024 22:46
A: globaleaks/globaleaks-whistleblowing-software ***@***.******@***.***>>
Cc: Matteo Perazzolo ***@***.******@***.***>>; Author ***@***.******@***.***>>
Oggetto: Re: [globaleaks/globaleaks-whistleblowing-software] Update globaleaks (Discussion #4292)


Thank you for your question.

GlobaLeaks is always released considering stability and security as top priority so that users should not have particular doubts when upgrading.
At the moment we are releasing a large set of minor releases following a very major change related to the client but we expect to be concluding soon with a relese 5.0.X concluding the release 5.0 as final.

Following this we will proceed developing and publishing a more precise release schedule with few major release a year (2 to 4) and minor releases issues only for security reasons.

Do you have any specific question ?
